pkg/kubelet/cm/containermap/container_map.go

```diff
@@ -20,23 +20,36 @@ import (
 	"fmt"
 )
 
-// ContainerMap maps (containerID)->(*v1.Pod, *v1.Container)
-type ContainerMap map[string]struct {
+type containerMapElem struct {
 	podUID        string
 	containerName string
+	running       bool
 }
 
+// ContainerMap maps (containerID)->(*v1.Pod, *v1.Container)
+type ContainerMap map[string]containerMapElem
+
 // NewContainerMap creates a new ContainerMap struct
 func NewContainerMap() ContainerMap {
 	return make(ContainerMap)
 }
 
 // Add adds a mapping of (containerID)->(podUID, containerName) to the ContainerMap
 func (cm ContainerMap) Add(podUID, containerName, containerID string) {
-	cm[containerID] = struct {
-		podUID        string
-		containerName string
-	}{podUID, containerName}
+	cm[containerID] = containerMapElem{
+		podUID:        podUID,
+		containerName: containerName,
+		running:       false,
+	}
```

Comment:
Nit: could call `AddWithRunningState()`
```suggestion
	AddWithRunningState(podUID, containerName, containerID, false)
```
to only update the elements in one place.